tutorials/fundamentals/array-toString.md

```diff
@@ -0,0 +1,33 @@
+To print an array in JavaScript, you can use the `toString()` method.
+The `toString()` method simply prints the elements of the array as a single string.
+
+```javascript
+const array = [1,2,3,4,5];
+
+array.toString(); // "1,2,3,4,5"
+```
+
+## Printing an Array in Node
+
+When using node.js, you must import the `util` module and use the `inspect()` function.
+This function will print the raw array as a string as shown below:
+
+```javascript
+const array = [1,2,3,4,5];
+const {inspect} = require('util');
+
+inspect(array); // [1,2,3,4,5]
```

Comment:
Worth showing what `inspect()` does to arrays of objects as well, one of the big benefits of `inspect()` over `toString()` is that `inspect()` pretty prints objects